tests/functional/fx_firstrun_v2_sign_up.js

```diff
@@ -36,28 +34,20 @@ define([
     },
 
     afterEach: function () {
-      var self = this;
-
       return this.remote
         .then(FunctionalHelpers.clearBrowserState())
-        .then(function () {
-          // ensure the next test suite (bounced_email) loads a fresh
-          // signup page. If a fresh signup page is not forced, the
-          // bounced_email tests try to sign up using the Sync broker,
-          // resulting in a channel timeout.
-          return self.remote
-            .get(require.toUrl(SIGNIN_URL))
-
-            .findByCssSelector('#fxa-signin-header')
-            .end();
-        });
+        // ensure the next test suite (bounced_email) loads a fresh
```

Comment:
This pattern is repeated over and over, I have another PR for the bounced_email suite to take care of this job itself, but didn't want to mix it in here.